e2e-tests/playwright.config.ts

```diff
@@ -178,13 +178,13 @@ export default defineConfig({
       command: 'cd .. && ./gradlew :api-admin:bootRun',
       url: 'http://localhost:8080/status',
       timeout: 180 * 1000,
-      reuseExistingServer: !CI
+      reuseExistingServer: true
```

Comment:
nice, so this means that we're always taking ownership of spinning up whatever resources we need outside the scope of the playwright tests.  That sounds right to me given the complexity of the setup